src/test/java/com/washingtonpost/arc/ans/v0_3/model/TestStory.java

```diff
@@ -19,13 +19,13 @@ String getSchemaName() {
 
     @Override
     Class getTargetClass() {
-        return Story.class;
+        return Collection.class;
     }
 
     @Test
     public void testStoryGood() throws Exception {
         testJsonValidation("story-fixture-good", true);
-        Story story = testClassSerialization("story-fixture-good");
+        Collection story = testClassSerialization("story-fixture-good");
```

Comment:
variable name should be 'collection' and fixture name should be 'collection-fixture-good'